swupd/swupd.go

```diff
@@ -412,7 +459,7 @@ func setMirror(swupdArgs []string, t string) (string, error) {
 func SetHostMirror(url string) (string, error) {
 
 	if urlErr := network.CheckURL(url); urlErr != nil {
-		return "", fmt.Errorf("Server not responding")
+		return "", fmt.Errorf(utils.Locale.Get("Server not responding"))
```

Comment:
Initially, I had concerns about having these translated in common code as this means the TUI will get translated strings that might now format right for other languages. But looking through the code base we have similar issues elsewhere, so we will not worry about this for now.